tasks/config.yml

```diff
@@ -22,6 +22,10 @@
   with_items:
     - "etc/apt/apt.conf.d/50unattended-upgrades"
 
+- name: Configuring APT timers
+  include_tasks: unattended_upgrades_timers.yml
+  when: apt_unattended_upgrades_download_timer_override | bool or apt_unattended_upgrades_upgrade_timer_override | bool
```

Comment:
Could you split this into two files and then use 
`when: apt_unattended_upgrades_download_timer_override | bool` 
& 
`when: apt_unattended_upgrades_upgrade_timer_override | bool`.

This will make the internal checks redundant.

Reply:
ok too